Source/Analytics/Read/Map/CaseReportsBeforeDay.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Dolittle.ReadModels;
+using Concepts;
+using Concepts.HealthRisk;
+using System.Collections.Generic;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Options;
+
+namespace Read.Map
+{
+    public class CaseReportsBeforeDay : IReadModel
+    {
+        public Day Id {get; set;}
+        
+        [BsonDictionaryOptions(DictionaryRepresentation.ArrayOfDocuments)]
+        public IDictionary<HealthRiskId, CaseReportsRetrieved> CasesPerHealthRisk { get; set; } 
```

Comment:
Should the name be CaseReportsPerHealthRisk to stick to the general namings in the project?